### ISSUE TITLE
is .app/cache really needed?

### ISSUE THREAD

**Issue by hamnis** [appmgr/appmgr#12]
Since this will be cached by the maven repo, is this really needed?

**Comment by trygvis** [comment]
Perhaps not. It was made to cover the case where you wanted to know that you could easily roll down to the previous version without relying on your Maven repository to be accessible. 

It can definitely be made into a configurable option. It could default to false so it is disabled.

**Comment by hamnis** [comment]
That would only apply if we didnt have the previous version unpacked. Since we never delete installed versions.

**Comment by trygvis** [comment]
Yes, that it true. The cache is useful for a "session" as it can call resolve multiple times (IIRC) so it would be useful to have some caching. But it can delete everything that is over one day old.
